.esbuild/serve.cjs

```diff
@@ -0,0 +1,53 @@
+const esbuild = require('esbuild');
+const http = require('http');
+const path = require('path');
+const { iifeBuild } = require('./util.cjs');
+
+// Start esbuild's server on a random local port
+esbuild
+  .serve(
+    {
+      servedir: path.join(__dirname, '..'),
```

Comment:
@aloisklink, there is a serve.js here, but it lacks the support for multiple directories of webpack.
<img width="484" alt="image" src="https://user-images.githubusercontent.com/10703445/189580684-5d7a421c-4bda-47d2-8d3e-8f661cf90fbd.png">

We might have to setup a reverse proxy as evanw suggested in https://github.com/evanw/esbuild/issues/796 to support the multiple files. 

I think we can bundle this with removing webpack + babel in another PR.

Reply:
:+1: Looks good to me.

We might also be able to just use `servedir: '..'` (rootdir), and just have everything use `./dist/mermaid.js` instead of `./mermaid.js`. But it's probably easier to stick that in another PR.

Reply:
The build is not the issue, the test html files are. 
Webpack serves all files in the 3 dirs at `/` so the cypress tests use the filename directly in the URL.

<img width="591" alt="image" src="https://user-images.githubusercontent.com/10703445/189585883-30a8caee-22b0-4314-bee9-bdd6c26de972.png">

So even if we change to rootdir, all the test files will have to be updated.

Reply:
> So even if we change to rootdir, all the test files will have to be updated.

:+1: 

Ouch, changing all the test files would be pretty painful. Hopefully we can use find+replace to partially automatically update them. Otherwise, it might be easier to setup a reverse proxy.